test/functional/test_host_io.py

```diff
@@ -111,15 +133,20 @@ def prepare_ssh(self):
 
     def setUp(cls):
         DOWNLOAD_URL = 'https://github.com/InfraSIM/test/raw/master/image/kcs.img'
-        MD5_KCS_IMG = 'cfdf7d855d2f69c67c6e16cc9b53f0da'
+        MD5_KCS_IMG = '986e5e63e8231a307babfbe9c81ca210'
         helper.fetch_image(DOWNLOAD_URL, MD5_KCS_IMG, test_img_file)
 
         cls.start_node()
         cls.port_forward()
 
+
     def tearDown(cls):
         if conf:
             cls.stop_node()
+        for i in range(97, 109):
```

Comment:
would be more readable with range(ord('a'), ord('z'))?

if there're more than 26 drives, identifier will grow to /dev/sdaa, /dev/sdab ... /dev/sdaaa, /dev/sdaab, ...etc